charts/fluent-bit/Chart.yaml

```diff
@@ -5,8 +5,8 @@ keywords:
   - logging
   - fluent-bit
   - fluentd
-version: 0.19.2
-appVersion: 1.8.9
+version: 0.19.3
+appVersion: 1.8.10
```

Comment:
```suggestion
appVersion: 1.8.9
```

app versino corresponds to https://fluentbit.io/announcements/

just the chart version needs a bump in this case